astropy/coordinates/baseframe.py

```diff
@@ -691,10 +695,21 @@ def isscalar(self):
         return self.has_data and self.data.isscalar
 
     @classmethod
-    def get_frame_attr_names(cls):
+    def get_frame_attr_defaults(cls):
+        """Return a dict with the defaults for each frame attribute"""
         return {name: getattr(cls, name)
                 for name in cls.frame_attributes}
 
+    @deprecated(
+        "5.2",
+        alternative="get_frame_attr_defaults",
+        message="If you only need the names, use frame_attributes.keys()"
```

Comment:
The `message` overwrites the default error message, so currently the users are not notified of the alternative they should be using.
```suggestion
        message="The {func}() {obj_type} is deprecated and may be removed in a future version. Use {alternative}() to obtain a dict of frame attribute names and default values. The fastest way to obtain the names is frame_attributes.keys()"
```